lib/internal/Magento/Framework/ObjectManager/Code/Generator/Repository.php

```diff
@@ -228,7 +228,6 @@ protected function _getGetMethod()
             'parameters' => [
                 [
                     'name' => 'id',
-                    'type' => 'int',
```

Comment:
> Fair enough - I'm not really interested in working on that so should I just close this PR?

It's up to you, in my opinion, this PR need to finalize. But anyway, you have done a good job and this task has very high priority for us because allows supporting PHP 7.x features and outdated dependencies are always a headache.